Tests/Mocks/StubCheckoutAPIService.swift

```diff
@@ -9,15 +9,22 @@
 import Checkout
 
 final class StubCheckoutAPIService: Frames.CheckoutAPIProtocol {
-  private(set) var createTokenCalledWith: (paymentSource: PaymentSource, completion: (Result<TokenDetails, TokenisationError.TokenRequest>) -> Void)?
-  func createToken(_ paymentSource: PaymentSource, completion: @escaping (Result<TokenDetails, TokenisationError.TokenRequest>) -> Void) {
-    createTokenCalledWith = (paymentSource, completion)
-  }
+    private(set) var createTokenCalledWith: (paymentSource: PaymentSource, completion: (Result<TokenDetails, TokenisationError.TokenRequest>) -> Void)?
+    func createToken(_ paymentSource: PaymentSource, completion: @escaping (Result<TokenDetails, TokenisationError.TokenRequest>) -> Void) {
```

Comment:
i think, functions should e after variables

Reply:
e after variables?